packages/http/test/typekit/http-request.test.ts

```diff
@@ -174,8 +174,7 @@ describe("HttpRequest Get Parameters", () => {
     const { createFoo } = (await runner.compile(`
       @test model Foo {
         @path(#{allowReserved: true}) id: string;
-        # suppress "deprecated" "Test"
-        @header(#{format: "csv"}) requestId: string[];
+        @header requestId: string[];
```

Comment:
i think that in theory should be the same as the other test above where you check `explode` is set